### DISCUSSION TITLE
メール内にMarkdownの記述ミス？

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#268] [要望・アイデア・不具合]
8月3日に送信されたランキングメールで`**[New]**`という文字列があったのですが、もしかするとMarkdownの記述ミスかな？と思いました。
ご確認よろしくお願いします。

<img width="529" alt="Screen Shot 2022-08-04 at 9 59 42" src="https://user-images.githubusercontent.com/1148320/182740732-5dacbe62-5ce8-4750-ac4e-9873bd1515ac.png">

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご連絡ありがとうございます！
担当チームに確認したところ記述ミスでした🙇
次回配信分より修正して配信いたします🙇💦教えてくださりありがとうございます！🙏
